From 821b37ef872cbc43bfd3b3c3d9b85aeebe863349 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Tue, 1 Nov 2011 00:28:37 +0100
Subject: ARM: gic: use module.h instead of export.h

The module.h cleanup series is not merged at this point, so use the
older header file for now, to make it build either way.

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
(cherry picked from commit 7e1efcf5d2039fb7a91e21df32f4175dbca4d61c)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 arch/arm/common/gic.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/common/gic.c b/arch/arm/common/gic.c
index 0ad3584..f4b2864 100644
--- a/arch/arm/common/gic.c
+++ b/arch/arm/common/gic.c
@@ -25,7 +25,7 @@
 #include <linux/init.h>
 #include <linux/kernel.h>
 #include <linux/err.h>
-#include <linux/export.h>
+#include <linux/module.h>
 #include <linux/list.h>
 #include <linux/smp.h>
 #include <linux/cpumask.h>
-- 
1.7.10.2.565.gbd578b5

